installscripts/dockerfiles/gitlab/gitlab.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+yes yes| /opt/gitlab/bin/gitlab-rake gitlab:setup RAILS_ENV=production GITLAB_ROOT_PASSWORD=gitlab12345 > ~/out.txt 2>&1
```

Comment:
Why this password hardcoded? Use a random value.

Reply:
The hard coded password is removed .